inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -737,6 +751,80 @@ void InsertCopyLayerPass::run() {
     }
 }
 
+void Concat4Dto2DPass::run() {
+    auto quantized = InferenceEngine::getInjectedData<QuantizedLayerParams>(pLayers->front());
```

Comment:
Please add comment describing what pattern you are trying to find and what transformation you are trying to do